moviebot/controller/server.py

```diff
@@ -1,64 +1,78 @@
+from moviebot.controller.controller_flask import ControllerFlask
 
 app = Flask(__name__)
-controller_telegram = ControllerTelegram()
-URL = "https://636d944311f9.ngrok.io/"  # Webhook url
+controller_flask = ControllerFlask()
 
 
-def bot_token():
-    """Gets bot token from config file."""
-    path = "config/bot_token.yaml"
-    with open(path, "r") as file:
-        config = yaml.load(file, Loader=yaml.Loader)
-        BOT_TOKEN = config["BOT_TOKEN"]
-        return BOT_TOKEN
-
-
-telegram_token = bot_token()
-bot = telegram.Bot(token=telegram_token)
+def run(config: Dict[str, Any]) -> None:
+    """Runs execute_agent in ControllerFlask and starts flask server.
 
+    Args:
+        config: Agent configuration.
+    """
+    controller_flask.execute_agent(config)
+    app.run(host="0.0.0.0", port=environ.get("PORT", 5001))
 
-@app.route("/{}".format(telegram_token), methods=["POST"])
-def respond():
-    """Receives to Telegram POST request."""
-    update = telegram.Update.de_json(request.get_json(force=True), bot)
-    text = update.message.text.encode("utf-8").decode()
 
-    if text == "/start":
-        controller_telegram.start(update, True)
-    elif text == "/restart":
-        controller_telegram.start(update, True)
-    elif text == "/help":
-        controller_telegram.help(update, True)
-    elif text == "/exit":
-        controller_telegram.exit(update, True)
+@app.route("/", methods=["GET", "POST"])
+def receive_message() -> None:
+    """Receives POST requests send from client."""
+    if request.method == "GET":
+        return "MovieBot is alive", 200
     else:
-        controller_telegram.continue_conv(update, True)
-    return "ok"
+        output = request.get_json()
+        print(output)
+        response = action(output)
+        if response:
+            return response
+        return "Message Processed"
 
 
-def set_webhook():
-    s = bot.setWebhook("{URL}{HOOK}".format(URL=URL, HOOK=telegram_token))
-    if s:
-        return "webhook ok"
-    else:
-        return "webhook failed"
+def action(output: Dict[str, Any]) -> Dict[str, Dict[str, str]]:
+    """Gets user id and payload from output and runs get_message in the
+    controller.
 
+    Args:
+        output: Output from request.
 
-def run(config):
-    """Runs execute_agent in ControllerTelegram and starts flask server.
+    Returns:
+        Object with message to send to the server.
+    """
+    event = output["entry"][0]["messaging"][0]
+    user_id = event["sender"]["id"]
+    controller_flask.initialize(user_id)
+    payload = get_message(output)
+    print(payload)
+    if payload is not None:
+        run_method_response = controller_flask.run_method(user_id, payload)
+        if run_method_response is True:
+            return controller_flask.send_message(user_id, payload)
+        elif run_method_response:
+            return run_method_response
+
+
+def get_message(output: Dict[str, Any]) -> str:
```

Comment:
consder using dataclasses here. It might be easier to align with front end development. 
See [dataclasses-json](https://pypi.org/project/dataclasses-json/).

Reply:
Note that I note sure what `quick_reply` and `postback` refer to. I was present in original file.